shared/chat/inbox/row/small-team/bottom-line.js

```diff
@@ -97,9 +99,10 @@ class BottomLine extends PureComponent<Props> {
           )
           break
         default:
-          snippetDecoration = this.props.snippetDecoration ? (
-            <Text type="BodySmall">{this.props.snippetDecoration}</Text>
-          ) : null
+          snippetDecoration =
+            this.props.snippetDecoration && !this.props.isTypingSnippet ? (
```

Comment:
Needs !!

Reply:
Ok, that’s been in there for a long time though. Is there anyway to get Flow to catch these?

Reply:
we've spent a little time on the flow thing and we haven't found a solution. we'd basically have to override the built in definition of react to disallow this (also this only really affects android which crashes out, ios/electron is fine with strings).
additionally i think this is slightly different than the old code. I think its a good idea to guard all these expressions anyway but in my thinking
old:
```
expr ? 'trueValue' : 'falseValue'
```
will, no matter what eval to 'trueValue' or 'falseValue' and nothing else
whereas the new expression
```
expr1 && expr2 ? 'trueValue' : 'falseValue'
```
is more ambiguous, depending of your knowledge of precedence rules.
aka on reading you might not know if its
```
expr1 && (expr2 ? 'trueValue' : 'falseValue')
// or
(expr1 && expr2) ? 'trueValue' : 'falseValue'
```
so looking at it, you might not be sure.

eitherway, ideally flow would catch this. 
in the meantime lets just be more explicit so its more clear